app/_locales/en/messages.json

```diff
@@ -6636,7 +6636,7 @@
     "message": "Decode smart contracts"
   },
   "use4ByteResolutionDescription": {
-    "message": "To improve user experience, we customize the activity tab with messages based on the smart contracts you interact with. MetaMask uses a service called 4byte.directory to decode data and show you a version of a smart contract that's easier to read. This helps reduce your chances of approving malicious smart contract actions, but can result in your IP address being shared."
```

Comment:
I thought that in order for the new message to be picked up correctly by the translation program (and in order to eventually be translated to other languages) you needed to create a new translation key and delete the old one, including deleting the old one on all languages. I might be wrong though, worth checking with the extension platform team.

Reply:
That's right, this key should be deleted and new one must be used.

Reply:
Done.